SConstruct

```diff
@@ -368,10 +368,9 @@ preForcedPhotCcd = command("forcedPhotCcd", [mapper, mergeMeasurements],
 forcedPhotCcd = [data.forced(env, tract=0) for data in sum(allData.values(), [])]
 
 
-gen3config = env.Command(os.path.join(REPO, "butler.yaml"), [mapper],
-                         [getExecutable("daf_butler", "makeButlerRepo.py") + " " + REPO])
-gen3repo = env.Command(os.path.join(REPO, "gen3.sqlite3"), [gen3config, forcedPhotCcd, forcedPhotCoadd],
-                       [getExecutable("ci_hsc", "gen3.py")])
+gen3repo = env.Command([os.path.join(REPO, "butler.yaml"), os.path.join(REPO, "gen3.sqlite3")],
+                       [forcedPhotCcd, forcedPhotCoadd],
```

Comment:
`mapper` was a dependency for `gen3config`, should it be added here too?

Reply:
`mapper` is also a dependency of `forcedPhotCcd` and `forcedPhotCoadd` (it's basically the first step in the Gen2 processing); I only included it explicitly before because the old `gen3config` target didn't actually depend on the later steps of the Gen2 pipeline, but the combined target of course does.